fedora_cloud_tests/azure.py

```diff
@@ -83,39 +84,30 @@ def _generate_test_log_path(self, image_definition_name):
             str: The run time name for the LISA tests.
         """
 
-        log_path, run_name = None, None
-
-        # Generate custom log if CUSTOM_LOG_PATH is set to True
-        if CUSTOM_LOG_PATH:
-            try:
-                base_log_path = os.path.expanduser("~/lisa_results")
-                os.makedirs(base_log_path, exist_ok=True)
-                _log.info("Base log path created: %s", base_log_path)
-
-                # Create image-specific log directory
-                log_path = os.path.join(base_log_path, image_definition_name)
-                os.makedirs(log_path, exist_ok=True)
-                _log.info("Log path created: %s", log_path)
-            except Exception as e: # pylint: disable=broad-except
-                _log.error("Failed to create log path: %s", str(e))
-                log_path = None
-        else:
-            _log.info("Using default log path")
-
-        # Generate custom run name if CUSTOM_RUN_NAME is set to True
-        if CUSTOM_RUN_NAME:
-            try:
-                current_date = datetime.now()
-                month_day = current_date.strftime("%B%d")
-                year = current_date.strftime("%Y")
-                time_str = current_date.strftime("%H%M")
-                run_name = f"{month_day}-{year}-{time_str}"
-                _log.info("Run name generated: %s", run_name)
-            except Exception as e:  # pylint: disable=broad-except
-                _log.error("Failed to generate run name: %s", str(e))
-                run_name = None
-        else:
-            _log.info("Using default run name")
+        run_name = None
+
+        # Create temporary directory to store the test results
+        try:
+            log_path = tempfile.mkdtemp(
+                prefix=f"lisa_results_{image_definition_name}",
+                suffix="_logs"
+            )
+            _log.info("Temporary log path created: %s", log_path)
+        except Exception as e:  # pylint: disable=broad-except
+            _log.error("Failed to create temporary log path: %s", str(e))
+            raise
+
+        # Create custom run name with current date and time stamp
+        try:
+            current_date = datetime.now()
+            month_day = current_date.strftime("%B%d")
+            year = current_date.strftime("%Y")
+            time_str = current_date.strftime("%H%M")
+            run_name = f"{month_day}-{year}-{time_str}"
```

Comment:
I'd recommend using UTC since otherwise it'll pick whatever the server local time is (which _should_ be UTC, but...). You can also use `strftime` directly. I'd recommend something that approximates ISO 8601:
```suggestion
            run_name = datetime.datetime.now(datetime.UTC).strftime("%Y-%m-%dT%H:%MZ")
```

Reply:
I have slight issue with using the suggested way. So, I used "run_name = datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%MZ")". Hopefully, it is similar to the way you suggested.